dpnp/dpnp_iface_mathematical.py

```diff
@@ -1544,6 +1548,64 @@ def negative(
     )
 
 
+def proj(
+    x,
+    /,
+    out=None,
+    *,
+    order="K",
+    where=True,
+    dtype=None,
+    subok=True,
+    **kwargs,
+):
+    """
+    Returns the projection of a number onto the Riemann sphere.
+
+    For all infinite complex numbers (including the cases where one component is infinite and the other is `NaN`),
+    the function returns `(inf, 0.0)` or `(inf, -0.0)`.
+    For finite complex numbers, the input is returned.
+    All real-valued numbers are treated as complex numbers with positive zero imaginary part.
+
+    Returns
+    -------
+    out : dpnp.ndarray
+    The projection of each element of `x`.
```

Comment:
You need to keep indentions for the second line, otherwise the documentation is generated with wrong formatting, like [here](https://intelpython.github.io/dpnp/pull/1535/reference/generated/dpnp.signbit.html) for `dpnp.signbit` and while [here](https://intelpython.github.io/dpnp/reference/generated/dpnp.add.html#dpnp.add) for `dpnp.add` shows as it has to.
```suggestion
    out : dpnp.ndarray
        The projection of each element of `x`.
```